README.rst

```diff
@@ -180,10 +180,8 @@ Getting Involved
 Join the conversation and help the community. Even if you do not plan to contribute
 to GraphAr itself or GraphAr integrations in other projects, we'd be happy to have you involved.
 
-- Join the mailing list: send an email to `graphar+subscribe@googlegroups.com <mailto:graphar+subscribe@googlegroups.com>`_.
-  Share your ideas and use cases for the project.
-- Join the `GraphAr Slack`_ channel.
-- Follow our activity on `GitHub issues <https://github.com/alibaba/GraphAr/issues>`_.
+- Ask questions on `GitHub Discussions`_. We welcome all kinds of questions, from beginner to advanced!
+- Follow our activity and ask for feature requests on `GitHub issues <https://github.com/alibaba/GraphAr/issues>`_.
```

Comment:
It's preferable to avoid adding a link here as we have already included it at the bottom of the page.

Reply:
Fixed